docs/build-tools/sbt.md

```diff
@@ -6,8 +6,27 @@ title: sbt
 sbt is most commonly used build tool in the Scala community works with Metals
 out-of-the-box.
 
+## Automatic installation
+
+Metals is able to automatically import sbt builds. When you open Metals in a new
+directory it will ask you to "Install via Bloop".
+
+![Import build](../assets/vscode-import-via-bloop.png)
+
+## Install without Bloop
+
+sbt does not implement the Metals uses the
```

Comment:
That sounds strange 😉

Reply:
sbt does not implement the Build Server Protocol that Metal uses ...